controller/handler_ctrl/circuit_request.go

```diff
@@ -59,7 +60,7 @@ func (h *circuitRequestHandler) HandleReceive(msg *channel.Message, ch channel.C
 				}
 			}
 			log = log.WithField("serviceId", service)
-			if circuit, err := h.network.CreateCircuit(h.r, id, service, logcontext.NewContext()); err == nil {
+			if circuit, err := h.network.CreateCircuit(h.r, id, service, logcontext.NewContext(), time.Now().UTC().Add(network.DefaultTimeout)); err == nil {
```

Comment:
Does using UTC() buy you anything? Since none of the dealines are leaving the system, not sure where it would make a difference.